includes/data-port/class-sensei-data-port-job.php

```diff
@@ -14,7 +14,7 @@
  * running data port tasks which are registered by subclasses.
  */
 abstract class Sensei_Data_Port_Job implements Sensei_Background_Job_Interface, JsonSerializable {
-	const OPTION_PREFIX = 'sensei-tools-';
+	const OPTION_PREFIX = 'sensei-tools-job-';
```

Comment:
Should the Data Cleaner class be cleaning up all of these on uninstall?

Reply:
We do this on deactivation by using method `cancel_all_jobs` as this is a job specific option and all jobs are cancelled on deactivation.